Makefile

```diff
@@ -5,7 +5,7 @@ IGNORE_A11Y = true
 include index.mk
 
 unit-test:
-	mocha --recursive test
+	mocha --recursive "./test/{,!(fixtures)/**/}*.js"
```

Comment:
Would the `*.js` regex alone be enough to avoid picking up files in the `fixtures` directory?

It could have its specificity further increased by making it look for the `*.test.js` suffix pattern (i.e. rather than just `*.js`) in case files with a `.js` extension end up in the `fixtures` directory or a `helpers` directory in the future.

It would also prevent us having to add a negation in the regex for any additional directories for non-test files added in the future, e.g. `helpers`.

Reply:
yes and no 😄 

Yes, i'll add further specificity but we'll still need to negate the `fixtures` directory as there are test files in there too.

Reply:
> we'll still need to negate the `fixtures` directory as there are test files in there too.

Do you mean for apps using `n-gage` rather than `n-gage` itself? Can you point me to an example please?

Having test files in `fixtures` directories seems like something we shouldn't be doing.

Reply:
`n-gage` itself has both unit and integration tests. The integration test part clones 3 FT repos (into `test/fixtures`) and runs make install to make sure theres no errors, but it doesnt cleanup/remove the cloned repos afterwards. As a result, if you run `make test` again, the mocha glob captures all the files inside the `test` folder, or it did before the fix on the PR.

![image](https://user-images.githubusercontent.com/102233/89190332-37bbf980-d599-11ea-8d6f-c18d491993e3.png)

Reply:
Blimey. And it would pick up any files within those repos that end `.test.js` which we won't want to run as part of the tests for `n-gage`.

I would not have expected an entire repo to be a fixture for a test but I suppose that makes sense given what `n-gage` does.